setup.cfg

```diff
@@ -33,6 +33,15 @@ zip_safe = False
 include_package_data = True
 install_requires =
     azure-monitor-opentelemetry-exporter == 1.0.0b6
+    opentelemetry-instrumentation == 0.32b0
+    opentelemetry-sdk == 1.12.0rc2
+    opentelemetry-api == 1.12.0rc2

```

Comment:
Added these to fix the lint issues such as `Unable to import 'opentelemetry.instrumentation.distro'` but it seems the issue was something else. Could you take a look at that lint error? @lzchen

Reply:
Removed